enterprise/backend/test/metabase_enterprise/data_editing/api_test.clj

```diff
@@ -61,16 +62,18 @@
                    (table-rows table-id))))
 
           (testing "PUT should update the relevant rows and columns"
-            (is (= {:updated [{:id 1, :name "Pidgey", :song "Join us now and share the software"}
-                              {:id 2, :name "Speacolumn", :song "Hold music"}]}
-                   (mt/user-http-request :crowberto :put 200 url
-                                         {:rows [{:id 1 :song "Join us now and share the software"}
-                                                 {:id 2 :name "Speacolumn"}]})))
-
-            (is (= [[1 "Pidgey" "Join us now and share the software"]
-                    [2 "Speacolumn" "Hold music"]
-                    [3 "Farfetch'd" "The land of lisp"]]
-                   (table-rows table-id))))
+            (is (= #{{:id 1, :name "Pidgey", :song "Join us now and share the software"}
+                     {:id 2, :name "Speacolumn", :song "Hold music"}}
+                   (set
```

Comment:
I'm surprised that order is non-deterministic. I think it should be part of the contract, so the FE can just use the same ordering.